src/Microsoft.Management.Deployment.CsWinRTProjection/Microsoft.Management.Deployment.CsWinRTProjection.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-windows10.0.22000.0;net8.0-windows10.0.22000.0</TargetFrameworks>
+    <Platform>AnyCpu</Platform>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <OutputPath>$(SolutionDir)$(Platform)\$(Configuration)\$(MSBuildProjectName)\</OutputPath>
+    <Configurations>Debug;Release;ReleaseStatic</Configurations>
+    <!--
+    Forcing this version to continue using an older CsWinRT release while issues are resolved.
+    !!! Remove this on the next Microsoft.Windows.CsWinRT package version update. !!!
+    -->
+    <WindowsSdkPackageVersion>10.0.22000.34</WindowsSdkPackageVersion>

```

Comment:
This should be .53 for .NET 8, and .52 for .NET 6

Reply:
Good reminder that there is a different number for 6 vs 8.  But we had to use an older one due to a bug; I can see if that bug has been fixed since.

Reply:
We can't use .34 anyway because it's not AOT compatible 😅
I suspect whatever that was, it should be fine on the latest versions.